lib/monotonic_tick_count.rb

```diff
@@ -53,5 +53,11 @@ class << self
     def now
       new(tick_count_f: Process.clock_gettime(Process::CLOCK_MONOTONIC))
     end
+
+    def timer
+      start = MonotonicTickCount.now
```

Comment:
If you need the class name, it's best to use `self.class` to keep it DRY. But in this case you're already in a class method. So just call `now`! :)